Zend/zend_compile.c

```diff
@@ -2884,6 +2885,16 @@ static zend_result zend_try_compile_cv(znode *result, zend_ast *ast) /* {{{ */
 			return FAILURE;
 		}
 
+		if (zend_string_equals_literal(name, "http_response_header")) {
+			if (type == BP_VAR_R && !CG(has_assigned_to_http_response_header)) {
```

Comment:
It would be better to check `!CG(has_assigned_to_http_response_header)` first to avoid a string comparison